tests/scripts/ask-redirect-test.sh

```diff
@@ -20,6 +20,8 @@ EXPECT CLOSE
 EXPECT CONNECT
 EXPECT ["GET", "foo"]
 SEND -ASK 12182 127.0.0.1:7402
+EXPECT ["GET", "foo"]
+SEND -ASK 12182 127.0.0.1:7402
 EXPECT CLOSE
```

Comment:
Why exactly do we need two ASK-redirects to a newly added node to trigger this bug?

I think it's possible to add comments between these EXPECT commands with a `#` prefix. (Maybe I'm wrong.)

Reply:
Added comment